src/index.js

```diff
@@ -58,14 +61,25 @@ const populateDOM = () => {
   const tasksList = document.querySelector('#tasks');
   let newList = '';
 
-  MyTasks.tasks.forEach((val) => {
```

Comment:
Your app is not performing as expected. I am unable to add a new task. Please find below the error displayed. Kindly fix it. As a result of the error, I am unable to checkout the status functionality. Kindly fix the error.

![image](https://user-images.githubusercontent.com/10332499/165926566-289004d5-f060-478a-8bef-1534e909d439.png)